ports/py-pyqt6/portfile.cmake

```diff
@@ -7,91 +8,59 @@ vcpkg_from_pythonhosted(
     SHA512          c2ff8c47c9b0e43c009d0c90f565a54344e6f384c67dd30c2d422465d0702c07713acc0095c8b67827d1146675611c07d548ba282a26e41bb60a0a21977a7a64
 )
 
-set(SIPBUILD_ARGS
-  "--confirm-license"
-  "--qmake" "${CURRENT_INSTALLED_DIR}/tools/Qt6/bin/qmake.exe" #TODO: append .exe only on windows
-  "--api-dir" "${CURRENT_PACKAGES_DIR}/share/qt6/qsci/api/python"
-  "--verbose"
-  "--qt-shared"
-  "--no-make"
-  "--disable" "QtDesigner"
-  "--pep484-pyi"
-  "--debug"
-  "--target-dir" "${CURRENT_PACKAGES_DIR}/lib/python${PYTHON3_VERSION_MAJOR}.${PYTHON3_VERSION_MINOR}/site-packages"
-  "--build-dir" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel-build"
-)
+if(CMAKE_HOST_WIN32)
+    set(EXE_SUFFIX ".exe")
+endif()
 
 set(SIPBUILD_ARGS
   "--confirm-license"
-  "--qmake" "${CURRENT_INSTALLED_DIR}/tools/Qt6/bin/qmake.exe" #TODO: append .exe only on windows
+  "--qmake" "${CURRENT_INSTALLED_DIR}/tools/Qt6/bin/qmake${EXE_SUFFIX}"
   "--api-dir" "${CURRENT_PACKAGES_DIR}/share/qt6/qsci/api/python"
   "--verbose"
   "--qt-shared"
   "--no-make"
   "--disable" "QtDesigner"
   "--pep484-pyi"
-  #  "--debug" # will create debuggable bindings
   "--target-dir" "${CURRENT_PACKAGES_DIR}/lib/python${PYTHON3_VERSION_MAJOR}.${PYTHON3_VERSION_MINOR}/site-packages"
-  "--build-dir" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel-build"
+  "--build-dir" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel"
 )
 
-message(STATUS "Running sipbuild... (${PYTHON3})")
-vcpkg_execute_required_process(
-        COMMAND "${PYTHON3}" "-m" "sipbuild.tools.build" ${SIPBUILD_ARGS}
-        WORKING_DIRECTORY "${SOURCE_PATH}"
-        LOGNAME "sipbuild-${TARGET_TRIPLET}-0"
-    )
-message(STATUS "Running sipbuild...finished.")
+message(STATUS "Running sipbuild...")
 
-# TODO: Can we use vcpkg_install_make here instead of duplicating code?
-set(MAKEFILE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel-build/Makefile")
+vcpkg_backup_env_variables(VARS PATH)
 
-vcpkg_list(SET make_opts)
-vcpkg_list(SET install_opts)
-set(path_backup "$ENV{PATH}")
-if (CMAKE_HOST_WIN32)
-    vcpkg_add_to_path(PREPEND "${SCRIPTS}/buildsystems/make_wrapper")
-    if(NOT DEFINED Z_VCPKG_MAKE)
-        vcpkg_acquire_msys(MSYS_ROOT)
-        find_program(Z_VCPKG_MAKE make PATHS "${MSYS_ROOT}/usr/bin" NO_DEFAULT_PATH REQUIRED)
-    endif()
-    set(make_command "${Z_VCPKG_MAKE}")
-    vcpkg_list(SET make_opts -j ${VCPKG_CONCURRENCY} --trace -f ${MAKEFILE})
+# Make sure sipbuild can find QtCore.dll
+vcpkg_add_to_path(PREPEND "${CURRENT_INSTALLED_DIR}/tools/Qt6/bin/" "${CURRENT_INSTALLED_DIR}/bin")
```

Comment:
`_HOST_` ?

Reply:
Will need to check again if that was for running host tools or to discover target dependencies. For the latter case it's probably the wrong approach.